lib/permutations.rb

```diff
@@ -1,4 +1,23 @@
-
+#Time complexity: O(n+m)
+#Space complexity: O(n)
 def permutations?(string1, string2)
```

Comment:
This will fail for `heelo` and `hello`.  You should instead count the number of times each letter occurs.